code/modules/client/preferences/species_features/vampire.dm

```diff
@@ -0,0 +1,18 @@
+/datum/preference/choiced/vampire_status
```

Comment:
Sure you don't want a default value on this?

Reply:
Sure but unsure on which is better

probably inoculated, simply to get everyone to at least try out the gimmick

Reply:
That was my thought, it's probably going to be fairly tougher to have enough vampires on after this